state/indexer/query_range.go

```diff
@@ -182,7 +197,7 @@ func conditionArg(c syntax.Condition) interface{} {
 	}
 	switch c.Arg.Type {
 	case syntax.TNumber:
-		return int64(c.Arg.Number())
+		return c.Arg.Number()
```

Comment:
Above, you did a similar conversion like this: `c.Arg.Number().Int64()`
Shouldn't these be consistent?

Reply:
This is only in places where we expect the argument to be the height. As the height is expected to be an `Int64`. This here will return `*big.Float`. then in functions where we expect it to return height, it will get the `int64` value using `.Int64()`.

Reply:
Makes sense